wiki/How_You_Can_Help!/en.md

```diff
@@ -1,80 +1,78 @@
 # How you can help!
 
-_This is a post that [peppy](/users/2 "peppy") made some time ago and requested to make it an article here. [You can see the original post here](/community/forums/topics/83155)._
+osu! itself has a large, and diverse community from all walks of life, and anyone can help make osu! more awesome for you, and for everyone.
```

Comment:
This paragraph could be longer for a better introduction to the article. Maybe say what the article does (and look at the current version to clarify why help from the community is greatly appreciated)